### PR TITLE
[processing] nodes -> vertices algorithm renaming

### PR DESCRIPTION
**Pull request by nirvn**
## Description
@nyalldawson , as part of the recent agreement on moving away from nodes and unify the terminology around vertex, this PR renames a few processing algorithm to use vertex/vertices.

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

- [x] Commit messages are descriptive and explain the rationale for changes
- [x] Commits which fix bugs include `fixes #11111` in the commit message next to the description
- [x] Commits which add new features are tagged with `[FEATURE]` in the commit message
- [x] Commits which change the UI or existing user workflows are tagged with `[needs-docs]` in the commit message and containt sufficient information in the commit message to be documented
- [x] I have read the [QGIS Coding Standards](https://docs.qgis.org/testing/en/docs/developers_guide/codingstandards.html) and this PR complies with them
- [x] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [x] New unit tests have been added for core changes
- [x] I have run [the `scripts/prepare-commit.sh` script](https://github.com/qgis/QGIS/blob/master/.github/CONTRIBUTE.md#contributing-to-qgis) before each commit

### CODE REVIEW

File: python/plugins/processing/algs/qgis/DensifyGeometries.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ class DensifyGeometries(QgisFeatureBasedAlgorithm):
     VERTICES = 'VERTICES'
 
     def tags(self):
-        return self.tr('add,vertices,points').split(',')
+        return self.tr('add,vertex,vertices,points').split(',')
```
Comment (nyalldawson) [inline python/plugins/processing/algs/qgis/DensifyGeometries.py]: We should probably add nodes here too, right?

**Review by nyalldawson — COMMENTED** [review]
You'll need to update the tests too

**Comment by nyalldawson** [comment]
+1

**Comment by nirvn** [comment]
@DelazJ , yeah, I'm usually not too bad at tagging this, slipped off my mind this time around 😄

**Comment by DelazJ** [comment]
I see, but it's not too late :)
Can you please open a ticket in docs? @ghtmtt is doing a big and iterative work on documenting the algs so it would be a pity not to have that information and complete eg https://github.com/qgis/QGIS-Documentation/pull/2256.

**Pull request merged**